Client/Frontend/Browser/Onboarding/OnboardingAdsCountdownViewController.swift

```diff
@@ -43,11 +43,12 @@ class OnboardingAdsCountdownViewController: OnboardingViewController, UNUserNoti
         contentView.resetAnimation()
         contentView.animate(from: 0.0, to: 1.0, duration: UX.animationTime) { [weak self] in
             guard let self = self else { return }
+            
+            //Show the confirmation screen no matter what..
+            self.contentView.setState(.finished)
```

Comment:
I thought this is supposed to be after a 1s delay?

Reply:
So it won't right after the animation?

Reply:
As seen in ticket, all the flows which show ads countdown have "1 second delay":

![](https://user-images.githubusercontent.com/2388823/66870504-6effe200-ef56-11e9-93b7-95e9e634d6ff.png)

Reply:
Hmm but this ticket doesn't make sense.. I can show it 1 second after the animation is complete, but you don't even notice this 1s at all because it's so fast so it's going to flash/flicker.

In any case, I added the 1s delay after the animation is complete.

Reply:
If that's the case, it's up to you to inform @karenkliu, showing her what it looks like in both scenarios and get a final decision and update the flows/spec if it changes, otherwise QA will just test based on spec and re-open ticket if it doesn't have the delay